main/src/main/scala/sbt/Defaults.scala

```diff
@@ -148,6 +148,9 @@ object Defaults extends BuildCommon {
     val m = (for (a <- cp; an <- a.metadata get Keys.analysis) yield (a.data, an)).toMap
     m.get _
   }
+  private[sbt] def sbtCredentialsEnv(): Option[Credentials] =
+    Option(System.getenv("SBT_CREDENTIALS")).map(raw => new FileCredentials(new File(raw)))
```

Comment:
minor nitpick: I'd prefer `sys.env.get("SBT_CREDENTIALS")` and also move it under https://github.com/sbt/sbt/blob/develop/main/src/main/scala/sbt/internal/SysProp.scala maybe.

Reply:
sure thing, we'll do later today. I used `System.getenv` because it's used into a other places and I figured that was a reason for its usage.

Do you want me to add a a commit that uses `sys.env` across the board (or in places where possible?)

Reply:
I've updated the as suggested: moved to Sysprop and used sys.env instead. I've also added a commit to use sys.env in the other places, if it's too risky, ill remove it.